bqetl_project.yaml

```diff
@@ -408,6 +408,16 @@ generate:
       firefox_desktop:
         # firefox_desktop is a single app containing multiple channels hence why set to null.
         channels: null
+      focus_android:
+        channels:
+        - release
+        - beta
+        - nightly
+      focus_ios:
```

Comment:
i didn't see any prerelease channels for focus ios, but its possible i missed them. do they actually exist?

Reply:
you are correct. Removed the other channels.